tests/unit/linalg/test_densematrix.cpp

```diff
@@ -14,6 +14,75 @@
 
 using namespace mfem;
 
+TEST_CASE( "DenseMatrix LinearSolve methods",
+           "[DenseMatrix]" )
+{
+   SECTION( "singular_system" )
+   {
+      constexpr int N = 3;
+
+      DenseMatrix A( N );
```

Comment:
Please see comment on spaces after and before parentheses...

Reply:
Done.